src/pybind/mgr/snap_schedule/fs/schedule_client.py

```diff
@@ -276,7 +276,7 @@ def create_scheduled_snapshot(self,
                     time = datetime.now(timezone.utc)
                     with open_filesystem(self, fs_name) as fs_handle:
                         snap_ts = time.strftime(SNAPSHOT_TS_FORMAT)
-                        snap_name = f'{path}/.snap/{SNAPSHOT_PREFIX}-{snap_ts}'
+                        snap_name = f'{path}/.snap/{SNAPSHOT_PREFIX}-{snap_ts}UTC'
```

Comment:
minor nit: use "_UTC" suffix

Reply:
... and think its probably a good idea to suffix with `%Z` which would use the tz the host/container is configured with.